### ISSUE TITLE
~/.config/phoenix/phoenix.js doesn't seem to be used

### ISSUE THREAD

**Issue by levitanong** [kasper/phoenix#204]
- Version: 2.6.1
- macOS: 10.13.4

I've made sure that neither
`~/.phoenix.js` nor `~/Library/Application Support/Phoenix/phoenix.js` exist.

**Comment by mafredri** [comment]
No problem, I recalled something similar having happened to me once upon a time :D.

Speaking of monitoring the logs, I usually follow them in the terminal via:

```console
$ log stream --process Phoenix
```

For me it's more convenient than firing up Console :)

**Comment by jasonm23** [comment]
`log stream --process` TIL.

I've been `tail -f`ing until now.

Thanks @mafredi

**Comment by levitanong** [comment]
Thanks, everyone! I think this issue can be closed now.

**Comment by kasper** [comment]
This f4f660ce6fa2a990581ac105f09658ad39ef832c should hopefully make the logging situation more clear.

**Comment by ezrafree** [comment]
I'm sorry to bring up an old issue here, but I ran into this exact same issue today.

I was logging with the `log stream --process Phoenix` command at the time, but none of the log output made it apparent to me that the configuration file at `~/.phoenix.js` that I'd just deleted was replaced by a blank file. From my perspective, everything just stopped working, causing me to think that for some reason being able to use a config at `~/.config/phoenix/phoenix.js` was broken for me.

I can't help but to wonder if the documentation could be further expanded to explain this specific situation a bit more. Perhaps something like:

> Please note: If you delete your main config file while Phoenix is still running, Phoenix will create a blank file in its place. Be sure and quit Phoenix when switching between using `~/.phoenix.js`, `~/Library/Application Support/Phoenix/phoenix.js` or `~/.config/phoenix/phoenix.js`.

If others agree, I'd be happy to submit a PR to the above effect.
